lib/house.rb

```diff
@@ -0,0 +1,49 @@
+class House
+  attr_reader :price, :address, :rooms, :details
+
+  def initialize(price, address)
+    @price = price[1..-1].to_i
+    @address = address
+    @rooms = []
+    @total_area = 0
+    @details = {"price" => @price,
```

Comment:
this is okay here, but as a matter of best practice, we don't create instance variables unless we have a good reason.  This hash can be created in a method with the same outcome, so we'd prefer to see it that way.